libs/interpret/src/lib/MLIDashboard/Controls/WhatIfTab/LocalImportancePlots.tsx

```diff
@@ -186,30 +187,29 @@ export class LocalImportancePlots extends React.Component<
                 topK={this.state.topK}
```

Comment:
is this related to issue https://github.com/microsoft/responsible-ai-toolbox/issues/1234 or something else?

Reply:
This is just to solve the alignment problem, currently toggle is set true but original data is not sort by abs, this default logic is not supported from the beginning.